pkg/credentials/credentials.go

```diff
@@ -254,6 +254,17 @@ func AppendCloudProviderVolumes(dpa *oadpv1alpha1.DataProtectionApplication, ds
 
 // add plugin specific specs to velero deployment
 func AppendPluginSpecificSpecs(dpa *oadpv1alpha1.DataProtectionApplication, veleroDeployment *appsv1.Deployment, veleroContainer *corev1.Container, providerNeedsDefaultCreds map[string]bool, hasCloudStorage bool) error {
+
+	init_container_resources := corev1.ResourceRequirements{}
+	if len(veleroDeployment.Spec.Template.Spec.Containers) > 0 {
+		for _, container := range veleroDeployment.Spec.Template.Spec.Containers {
+			if container.Name == "velero" {
+				init_container_resources = container.Resources
+				break
+			}
+		}
+	}
```

Comment:
Here's another heart breaking thought..
```suggestion
	init_container_resources := veleroContainer.Resources
```

Reply:
sorry noticed just now.

Reply:
We kind of do this logic already here https://github.com/openshift/oadp-operator/blob/090604ba0304cfe37ffbec53c9dc9f8f14bf4424/controllers/velero.go#L442-L448